app/elements/devguide-shell.html

```diff
@@ -216,17 +269,42 @@
     },
 
     attached: function() {
+      Polymer.RenderStatus.afterNextRender(this, function() {
+        // Somebody might look at the API docs. Be ready.
+        this.importHref(this.resolveUrl('/elements/iron-doc-viewer.html'));
+      });
+
       // Compute the active page from the parent pw-shell's app route.
       var pwShell = this.parentElement;
       if (pwShell) {
         var path = pwShell._sectionRoute.path;
+
         // If it ends in a /, then it's a top level path like /1.0/tools/,
         // so add an index to it.
         if (path[path.length - 1] === '/') {
           path += 'index';
         }
         this.activePage = path;
       }
+
+      // Figure out which version we're displaying, 1.0 or 2.0. You can't rely on
+      // the parent's version, because if we clicked on the "see 2.0" <a> link inside
+      // this page, then the pw-shell would not have refreshed, so it doesn't
+      // know the version has changed.
+      pwShell.updateDocsVersion();
+      this._setActiveDocsVersion(pwShell.activeDocsVersion);
+      this.$.versionSelect.selected = this.activeDocsVersion == '1.0' ? 0 : 1;
+
+      // Update the version when the version selector is used.
+      this.$.versionSelect.addEventListener('iron-select', function(event) {
```

Comment:
Suggest binding event listener using Polymer-style attributes (to cleanup `attached` callback):

```html
<paper-listbox on-iron-select="_listboxVersionChanged">
```